### DISCUSSION TITLE
Pedalino with 15 footswitch to Ampero Stomp II

### DISCUSSION THREAD

**Discussion by regisampa** [alf45tar/PedalinoMini#515] [Show and tell]
Good morning my friends.

Finally I managed to build my Pedalino Mini to control my Ampero Stop II. The possibilities are countless.

There are 15 footswitches in a 263mm by 183mm ABS box purchased on Aliexpress.

I really liked the result.

Hugs

![IMG_20240515_213610](https://github.com/alf45tar/PedalinoMini/assets/168490796/97dab301-aa6c-44ea-b7a3-81fe1f5fb71f)
![IMG_20240515_213733](https://github.com/alf45tar/PedalinoMini/assets/168490796/e56dbf7e-09b9-435e-9786-34dc7234460e)
![IMG_20240515_213351_1](https://github.com/alf45tar/PedalinoMini/assets/168490796/e563f030-a967-4cb6-ac78-addd92c9f948)
![IMG_20240515_213547](https://github.com/alf45tar/PedalinoMini/assets/168490796/2737da52-6488-47e2-81e9-4d5d59a08fad)
![IMG_20240515_213511](https://github.com/alf45tar/PedalinoMini/assets/168490796/b240364d-b82f-486b-b8e3-7eeb1a58452d)

**Comment by upgrader-dev** [comment]
Awesome congratulations. How did you mount the board on the ABS case? Can you send pictures of the inside to see how you managed all these wires?

**Comment by regisampa** [comment]
Hi.

For now I installed the board with double-sided tape at the bottom of the box and it is well fixed, I used five buttons as ladder in a vertical configuration so I can activate the command of buttons simultaneously, I wouldn't be able to use buttons  simultaneously if the ladder configuration was on horizontal.

Thanks.

**Comment by dimarzio49** [comment]
Hello. I also assembled a controller for Ampero Stomp 2. Only with 6 buttons. I have a question. You use the Toggle CC command to control slots or turn a tuner or looper on/off. If yes how did you do it. I can't find the Toggle SS combination. Firmware 3.5

**Comment by regisampa** [comment]
Hi my friend.

To access a specific preset I use (Action 1: ON: Press, Send: Control Change, Control: 1, Chanel: 1, CC: 0, Value 1 / Action 2: ON: Press, Send: Program Change, Control: 1, Chanel: 1, PC: 27) in this example I access PRESET A53-2.

To activate the tuner I created a sequence (Sequence 1 / Line 1, Send: Control Change, Chanel: 1, CC: 60, Value: 127, LED: Default, / Line 2, Send: Control Change, Chanel: 1, CC: 60, Value: 0, LED: Default) Then I created a control (ON: Press, Send: Step bt Step, Control: 21, Sequence: 1, Step: 1, Tag 1 Value 0, Tag 2 Value 127), this way the tuner activates and deactivates when I press the pedal.

![image](https://github.com/user-attachments/assets/df029b7d-b8db-4ec0-8314-23949828874c)

![image](https://github.com/user-attachments/assets/71dc7e3c-fc81-4555-9831-784fe32cae0e)

![image](https://github.com/user-attachments/assets/36177649-b1d4-4d13-acea-ac24702de161)
